engine/src/main/java/org/camunda/bpm/engine/ProcessEngineConfiguration.java

```diff
@@ -259,7 +258,9 @@ public abstract class ProcessEngineConfiguration {
   protected Object jpaEntityManagerFactory;
   protected boolean jpaHandleTransaction;
   protected boolean jpaCloseEntityManager;
+
   protected int defaultNumberOfRetries = JobEntity.DEFAULT_RETRIES;
+  protected int historyCleanupDefaultNumberOfRetries = JobEntity.DEFAULT_RETRIES;
```

Comment:
💭 We can document the configuration property in https://docs.camunda.org/manual/develop/reference/deployment-descriptors/tags/process-engine/#history-cleanup-configuration-parameters

Reply:
PR for updating documentation can be found [here](https://github.com/camunda/camunda-docs-manual/pull/1471).